modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/http/access/AccessLogger.java

```diff
@@ -190,19 +202,50 @@ protected synchronized void open() {
 
         // Open the current log file
         try {
-            String pathname;
+            Date today = new Date(System.currentTimeMillis());
+            String pathname = dir.getAbsolutePath() + File.separator + AccessConstants.getPrefix() +
```

Comment:
Line wrapping needs to happen before an operator, shall we correct in all places?

Reply:
+1 filed